src/lib.rs

```diff
@@ -18,6 +18,8 @@
 ///     finish_data(&a.bytes);
 /// }
 /// ```
+
+#[cfg(feature = "std")]
```

Comment:
Maybe better is to have `!feature = "no_std`.

Reply:
I think this looks ok.